README.rst

```diff
@@ -81,7 +81,7 @@ provider::
 
     [dependencies]
     ndarray = { version = "0.13.0", features = ["blas"] }
-    blas-src = { version = "0.2.0", default-features = false, features = ["openblas"] }
+    blas-src = { version = "0.4.0", default-features = false, features = ["openblas"] }
```

Comment:
We should also bump `openblas-src` to `0.7` :pray:

Reply:
Done.